docs/changelog.md

```diff
@@ -67,6 +67,11 @@ permalink: /changelog/
     * Rename mozac_ic_cookies to mozac_ic_cookies_24
     * Rename mozac_ic_sync to mozac_ic_sync_24
 
+# 117.0
+
+* **concept-engine**
+  * 🌟️️ Add `ProductUrlStatus` to `SessionState` instance to indicate the status of the given session state see more on [bug #1842638](https://bugzilla.mozilla.org/show_bug.cgi?id=1842638).
```

Comment:
Could be more specific: `indicate whether or not a product page of a given session state is being displayed`